lib/ble_manager.dart

```diff
@@ -137,6 +137,11 @@ abstract class BleManager {
   ///
   /// If [serviceUUIDs] is empty, this will return an empty list.
   Future<List<Peripheral>> connectedPeripherals(List<String> serviceUUIDs);
+
+  /// Creates a peripheral which may not exist. Since the [peripheralId] might
+  /// be a UUID or a MAC address, depending on the platform, its format is
+  /// not validated.
+  Peripheral createUnsafePeripheral(String peripheralId, {String name});
```

Comment:
Maybe add an information that 'peripheralId' passed here is meant to be only the `peripheralId` of a previously scanned peripheral on this particular device — as a general rule 'peripheralId' is valid only on the same device it was scanned.